configure.py

```diff
@@ -2669,6 +2668,9 @@ def find_canonical_os_name(os_name_variant):
     if options.via_amalgamation:
         raise UserError("--via-amalgamation was removed. Use --amalgamation instead.")
 
+    if options.destdir:
+        raise usererror("--destdir was removed. Use the DESTDIR environment variable instead.")
```

Comment:
usererror -> UserError

I think it it worth mentioning that DESTDIR is relevent for the installation step only, e.g.

> --destdir was removed. Instead use the DESTDIR environment variable when running make install.